scripts/sdf-glyph-foundry/0.1.1/.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: generic
+sudo: false
```

Comment:
remove this `sudo:false` to avoid odd travis behavior of also publishing via `osx`. For header-only builds we need only one publish job